mubench.pipeline/tasks/implementations/findings_filters.py

```diff
@@ -2,13 +2,13 @@
 from typing import List
 
 from data.detector_run import DetectorRun
-from data.finding import SpecializedFinding, Finding
+from data.finding import Finding
 from data.misuse import Misuse
 
 
 class PotentialHits:
-    def __init__(self, findings: List[SpecializedFinding]):
-        self.findings = findings
+    def __init__(self, findings: List[Finding], detector_run: DetectorRun):
+        self.findings = detector_run.detector.specialize_findings(detector_run.findings_path, findings)
```

Comment:
I'm torn whether it makes sense to do the conversion here or in the publish task... after all, we do the conversion for the publication. The currently solution works as well, but it feels a little less explicit. What do you think?

Reply:
I've just added it here since it was a small change and the publish findings task seemed pretty loaded already. However, since we've removed it by accident this time... that could probably happen again. I guess it's less likely to be overlooked if we put it where the upload is prepared anyway.